### PR TITLE
Add a preview playground redirect page

### PR DESCRIPTION
**Pull request by j-f1**
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

This should allow the PR template to link to https://prettier.io/playground-redirect and have it redirect to the appropriate preview playground.

### CODE REVIEW

**Comment by j-f1** [comment]
Test: https://deploy-preview-5321--prettier.netlify.com/playground-redirect

File: website/pages/playground-redirect.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <h1>Redirecting…</h1>
+  <p>Redirecting you to the appropriate playground. If the redirect doesn’t work, <a href="javascript:history.back()">go back to the PR</a> and click the appropriate link.</p>
+  <script>
+    const match = /^https:\/\/github\.com\/prettier\/prettier\/pull\/(\d+)/.exec(document.referrer)
+    if (match != null) {
+      const [/* url */, pr] = match
+      location.replace(`https://deploy-preview-${pr}--prettier.netlify.com/playground`)
+    } else {
+      const el = document.createElement('p')
+      el.textContent = `The referrer URL (${document.referrer}) was not recognized. Sorry :(`
```
Comment (ikatyang) [inline website/pages/playground-redirect.html]: It'd be better to also guide people to find the link here I guess:

image

**Comment by ikatyang** [comment]
Update the PR template here as well?

File: website/pages/playground-redirect.html (hunk 1 above)
Comment (j-f1) [reply website/pages/playground-redirect.html]: Do you have any suggestions for the wording?

Comment (ikatyang) [reply website/pages/playground-redirect.html]: A gif should be enough:

(from https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/)

**Comment by lipis** [comment]
I formatted it with our latest HTML formatter... I hope it's fine :)

File: website/pages/playground-redirect.html (hunk 1 above)
Comment (j-f1) [reply website/pages/playground-redirect.html]: Added. Sorry for the delay.

**Comment by lipis** [comment]
Merginnn.. I just want to see this list to get smaller :)

https://github.com/prettier/prettier/milestone/21

**Pull request merged**